tests/materials-page/materials-page.query.test.js

```diff
@@ -29,6 +30,15 @@ describe('Materials-about block queries', () => {
     expect(blocks.items.length).toBe(1);
   });
 
+  it('Should get materials-bout blocks by type', async () => {
+    const blocks = await getMaterialsBlocksByType(
+      { type: 'main', limit: 0, skip: 0, filter: { search: 'ti' } },
+      operations
+    );
+
+    expect(blocks.items.length).toBe(0);
```

Comment:
you added new material with fields title: 'title', type: 'bottom',  before all. Maybe you should filter response by it? 
So you won't get an empty array in the response.

Reply:
Yes, surely it is valid remark. And it will be easier  to understand if "it" section will match to what we expect to get. I added some new tests to this query so you can  also review them.